doc/developers/advanced_installation.rst

```diff
@@ -193,22 +193,12 @@ Installing build dependencies
 Linux
 -----
 
-Installing from source requires you to have installed the scikit-learn runtime
-dependencies, Python development headers and a working C/C++ compiler.
-Under Debian-based operating systems, which include Ubuntu::
+Installing from source without conda or pip requires you to have installed the
+scikit-learn runtime dependencies, Python development headers and a working
+C/C++ compiler. Under Debian-based operating systems, which include Ubuntu::
```

Comment:
I'm not sure to get the purpose of this section. It seems to give the dependencies if you want to build without pip or conda. But everywhere else we only explain how to install everything with pip.

Reply:
This is to build from source. pip can't install python headers or compilers. If you want to build from source without conda you have to install compiler and headers.
Listing the python packages here is a bit strange, though. I think that is to avoid having to build them, these instructions predate binary wheels.

Reply:
So I think part of this can be removed, but we should have instructions on how to build without conda.

Reply:
Ok I see. I updated a bit. Let me know what you think